### PR TITLE
[AAP-6237]: Add ansible-rulebook CLI documentation

### PR DESCRIPTION
**Pull request by abaiken**
[AAP-6237:](https://issues.redhat.com/browse/AAP-6237)
Documentation: update docs/usage.rst

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -3,19 +3,52 @@ Usage
...
+
+    ansible-rulebook --version
+
 The normal method for running `ansible-rulebook` is the following::
 
     ansible-rulebook --inventory inventory.yml --rules rules.yml
 
-If you are using custom event source plugins use the following::
+.. note::
+    The `--rules` option requires the `--inventory` option.
```
Comment (Alex-Izquierdo) [inline docs/usage.rst:47]: `--rules` flag has been changed to `--rulebook`
Ref: https://github.com/ansible/ansible-rulebook/pull/189

**Review by Alex-Izquierdo — CHANGES_REQUESTED** [review]

File: docs/usage.rst [hunk 2]
```diff
@@ -3,19 +3,53 @@ Usage
...
+======================
+CLI Usage and Examples
+======================
+
+
+The `ansible-rulebook` CLI supports the following options::
+
+    Options:
+    -h, --help                  Show this page
+    -v, --vars=<v>              Variables file
```
Comment (Alex-Izquierdo) [inline docs/usage.rst:18]: Duplicated, `-v`is not supported yet

**Review by Alex-Izquierdo — CHANGES_REQUESTED** [review]

File: docs/usage.rst [hunk 3]
```diff
@@ -3,19 +3,60 @@ Usage
...
 
+If you are using custom event source plugins use the following::
 
     ansible-rulebook --inventory inventory.yml --rulebook rules.yml -S sources/
 
-Here `sources` is a directory containing your event source plugins.
+.. note::
+    Here `sources` is a directory containing your event source plugins.
+
+If `eda-server` is running `ansible-rulebook` the `--worker`, `--id`, and `--websocket-address` options should be used::
```
Comment (Alex-Izquierdo) [inline docs/usage.rst:54]: I would remove any mention to EDA-server, as it can be used independently. I would just describe that does the flags.

Comment (abaiken) [reply docs/usage.rst:54]: I can do that but I thought these flags were only used when also using `eda-server`. Are there use cases where these specific flags are used without `eda-server`?

Comment (Alex-Izquierdo) [reply docs/usage.rst:54]: Well, websocket is a way to expose the event stream data, can be used in multiple ways.  

Anyway, even if those flags are intended for eda-server as this project is independent, I only would explain what they do. What do you think @benthomasson?

**Review by Alex-Izquierdo — DISMISSED** [review]
Minor comment, otherwise LGTM

**Comment by mkanoor** [comment]
@abaiken The cli needs JAVA_HOME env var to be set like mentioned in the code here

https://github.com/ansible/ansible-rulebook/blob/310ec2414d96f60a295645b514c52a23a56cc9eb/ansible_rulebook/cli.py#L125

We should add a note about this, we need Java to be installed and JAVA_HOME set.

**Comment by Alex-Izquierdo** [comment]
> @abaiken The cli needs JAVA_HOME env var to be set like mentioned in the code here
> 
> https://github.com/ansible/ansible-rulebook/blob/310ec2414d96f60a295645b514c52a23a56cc9eb/ansible_rulebook/cli.py#L125
> 
> We should add a note about this, we need Java to be installed and JAVA_HOME set.

That is covered here: https://github.com/ansible/ansible-rulebook/pull/196

File: docs/usage.rst (hunk 3 above)
Comment (abaiken) [reply docs/usage.rst]: @Alex-Izquierdo thank you for the explanation! I removed the references to `eda-server` - take another look and make sure what I did makes sense!

**Review by Alex-Izquierdo — APPROVED** [review]
LGTM

**Review by ddonahue007 — APPROVED** [review]
lgtm

**Review by shannon-donahue — APPROVED** [review]
lgtm

**Pull request merged**